langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockAnthropicStreamingChatModel.java

```diff
@@ -30,4 +41,36 @@ public enum Types {
             this.value = modelID;
         }
     }
+
+    @Override
+    public void generate(final UserMessage userMessage, final StreamingResponseHandler<AiMessage> handler) {
+        List<ChatMessage> sanitizedMessages = sanitizeMessages(List.of(userMessage));
```

Comment:
does it make sense to sanitize single user message?

Reply:
removed